packages/info/lib/diagnoses/config.js

```diff
@@ -2,23 +2,23 @@
 
 const chalk = require('chalk');
 
-module.exports = class ConfigDiagnosisMixins {
+module.exports = class ConfigValidationMixin {
   constructor(config) {
     this.config = config;
-    this.diagnoses = [];
+    this.warnings = [];
   }
-  diagnoseConfig() {
-    const { _warnings: diagnoses } = this.config;
-    this.diagnoses = diagnoses;
+  validateConfig() {
+    const { _warnings } = this.config;
+    this.warnings = _warnings;
   }
-  logDiagnoses(logger) {
-    const { diagnoses } = this;
-    if (diagnoses.length) {
-      const warnings = diagnoses.map(
-        (diagnosis) => `${chalk.yellow('-')} ${diagnosis}`
-      );
+  logMessages(logger) {
```

Comment:
I would expect `logWarnings` but we used this syntax for all the rest.

Reply:
It is not limited to warnings, it could be errors, warnings, infos or something else, therefore I named it `logMessages`

Reply:
How about `logResults`?

Reply:
I've gone with `logResults`